### PR TITLE
Add gburges to cluster-wide.yaml

### PR DESCRIPTION
**Pull request by gburges**

### CODE REVIEW

**Comment by gburges** [comment]
@kargakis pls review

File: projects/azure/cluster-wide.yaml [hunk 1]
```diff
@@ -24,3 +24,4 @@ items:
   - pweil-
   - thekad
   - y-cote
+  - gburges
```
Comment (kargakis) [inline projects/azure/cluster-wide.yaml]: Can you keep this sorted like the rest of the entries?

Comment (gburges) [reply projects/azure/cluster-wide.yaml]: done

**Comment by kargakis** [comment]
Updated manually in the CI cluster, you should have access in the `azure` namespace now
```
$ oc apply --as=system:admin -f projects/azure/cluster-wide.yaml 
group.user.openshift.io/azure-team configured
Error from server (AlreadyExists): project.project.openshift.io "azure" already exists
```

**Pull request merged**